sass/patternfly-react/_pagination.scss

```diff
@@ -0,0 +1,16 @@
+.content-view-pf-pagination {
+  .form-group {
+    .pagination-pf-pagesize.btn-group {
+      display: flex;
+      float: none;
+      margin-left: 0;
+      margin-right: 5px;
+      margin-bottom: 0;
+      padding-right: 13px;
+      width: auto;
+    }
+    .dropdown-menu {
+      min-width: auto;
+    }
+  }
+}
```

Comment:
Is this necessary with the update to Patternfly in https://github.com/patternfly/patternfly/pull/910

Reply:
done